lib/run-status.js

```diff
@@ -48,7 +48,9 @@ class RunStatus extends Emittery {
 			skippedTests: 0,
 			todoTests: 0,
 			uncaughtExceptions: 0,
-			unhandledRejections: 0
+			unhandledRejections: 0,
+			runOnlySelectedLines: false,
```

Comment:
I know I added it, but with regards to the other properties, is this in correct tense?

Reply:
Yea I'll rename.